watertap/property_models/cryst_prop_pack.py

```diff
@@ -1141,13 +1141,15 @@ def calculate_scaling_factors(self):
         # Scaling material heat capacities
         if self.is_property_constructed('cp_solute'):
             for p in ['Sol', 'Liq']:
-                if iscale.get_scaling_factor(self.cp_solute) is None:
-                    iscale.set_scaling_factor(self.cp_solute[p], iscale.get_scaling_factor(self.cp_phase['Liq']))
+                if iscale.get_scaling_factor(self.cp_solute[p]) is None:
+                    iscale.set_scaling_factor(self.cp_solute[p], iscale.get_scaling_factor(self.cp_phase['Liq'], 
+                        default=1, warning=True))
 
         if self.is_property_constructed('cp_solvent'):
-            for p in ['Vap', 'Liq']:
-                if iscale.get_scaling_factor(self.cp_solvent) is None:
-                    iscale.set_scaling_factor(self.cp_solvent[p], iscale.get_scaling_factor(self.cp_phase['Liq']))
+            for p in ['Liq', 'Vap']:
+                if iscale.get_scaling_factor(self.cp_solvent[p]) is None:
+                    iscale.set_scaling_factor(self.cp_solvent[p], iscale.get_scaling_factor(self.cp_phase['Liq'], 
+                        default=1, warning=True))
 
```

Comment:
Revisit why we need to do this in a subsequent PR

Reply:
Addressed. Supplied the scaling factor directly rather than calling get_ecaling_factor